pkgs/applications/misc/zathura/core/default.nix

```diff
@@ -24,8 +24,9 @@ stdenv.mkDerivation (finalAttrs: {
     "-Dmanpages=enabled"
     "-Dconvert-icon=enabled"
     "-Dsynctex=enabled"
+    "-Dtests=disabled"
```

Comment:
Could add a comment above explaining why tests are disabled

Reply:
I don't know what the problem is with libseccomp when testing
 could be here [#324](https://git.pwmt.org/pwmt/zathura/-/issues/324)